### PR TITLE
Adding Windows integration to test policy

### PR DESCRIPTION
**Pull request by dikshachauhan-qasource**
## What does this PR do?
Code added for adding Windows integration to fleet suite.

## Why is it important?
As one of the core integration, so checking out, if integration is available and added to policy without any errors.

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the Unit tests (`make unit-test`), and they are passing locally
- [x] I have run the End-2-End tests for the suite I'm working on, and they are passing locally
- [ ] I have noticed new Go dependencies (run `make notice` in the proper directory)

### CODE REVIEW

**Review by pazone — APPROVED** [review]

File: e2e/_suites/fleet/system_integration.go [hunk 1]
```diff
@@ -100,6 +100,14 @@ func (fts *FleetTestSuite) thePolicyIsUpdatedToHaveSystemSet(name string, set st
 		}
 
 		kibanaInputs = metricsInputs(name, set, "/linux_metrics.json", metrics)
+	} else if name == "winlog" {
```
Comment (pazone) [inline e2e/_suites/fleet/system_integration.go:103]: Indentation

**Comment by cachedout** [comment]
@dikshachauhan-qasource Are you waiting on anything else from us on this one?

**Comment by dikshachauhan-qasource** [comment]
Hi @cachedout,

We are pending with one reviewer @kuisathaverat. Also, I don't have permissions to merge code to the main branch, so that's why needed help from onsite team to merge once review process is completed.

**Comment by cachedout** [comment]
@kuisathaverat Can you please swing by here and review this PR when you have a moment?

**Comment by kuisathaverat** [comment]
There are a couple of liting issues see https://github.com/elastic/e2e-testing/actions/runs/4053562039/jobs/6974323631
The Go code is not correctly indented see https://github.com/elastic/e2e-testing/pull/3361?notification_referrer_id=NT_kwDOAFJo1LI1Mjg4MTY0MDU5OjU0MDA3ODg#discussion_r1073420560

File: .ci/.e2e-tests-daily.yaml [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
+#      - name: "Fleet"
+#       tags: "fleet_mode"
+#       platforms: ["ubuntu_22_04_amd64"]
 #         platforms: ["centos8_arm64", "centos8_amd64", "debian_10_arm64", "debian_10_amd64", "sles15", "debian_11_amd64", "ubuntu_22_04_amd64", "windows2019"]
-#       - name: "Integrations"
-#         tags: "integrations"
-#         platforms: ["debian_11_amd64"]
+       - name: "Integrations"
+         tags: "integrations"
+         platforms: ["debian_10_amd64"]
```
Comment (kuisathaverat) [inline .ci/.e2e-tests-daily.yaml:23]: The change in daily scenarios does not seem related to the changes. Is it correct?

Comment (dikshachauhan-qasource) [reply .ci/.e2e-tests-daily.yaml:23]: Hi Ivan,

No its not related to the current changes, Just to run the tests at my end and validate the changes as working fine.

**Comment by cachedout** [comment]
@dikshachauhan-qasource Looks like there is still a lint issue here: https://github.com/elastic/e2e-testing/actions/runs/4053562039/jobs/6974323631

**Comment by pazone** [comment]
@kuisathaverat According to the linter we need trailing lines in `*.json` and `*.feature` files. Shouldn't we add an exclusion for those types of files?

**Comment by kuisathaverat** [comment]
> Shouldn't we add an exclusion for those types of files?

no, those files must be trailing like other files

**Comment by dikshachauhan-qasource** [comment]
Closing PR in favor of https://github.com/elastic/e2e-testing/pull/3422
